FireCARES/firecares
# Feature/ems csv prediction

## About

- Adds support for `ems` column to `import-predictions` command 
- Adds EMS Prediction to Department detail template
- Adds ems to the model and generated migrations
- Fixes issue when strings are mixed with numbers in prediction CSV.

<img width="335" alt="Screen Shot 2019-04-08 at 10 31 29 AM" src="https://user-images.githubusercontent.com/4162671/55736732-854a4580-59e9-11e9-93e5-3ce4f8d667cc.png">

<img width="1167" alt="Screen Shot 2019-04-05 at 7 17 45 PM" src="https://user-images.githubusercontent.com/4162671/55736034-2afcb500-59e8-11e9-97a3-18c4869af295.png">

## Relevant Issues

- Closes https://github.com/FireCARES/firecares/issues/515

## Testing Instructions

1. Run `./manage.py import-predictions scripts/ems.csv`
2. Go to a department with EMS prediction data (http://192.168.33.15:8000/departments/87255/los-angeles-county-fire-department) and check if prediction is shown
